### DISCUSSION TITLE
Scrum Prep Milestone 1 -Week 2

### DISCUSSION THREAD

**Discussion by AngelUPR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#44] [General]
In one or two sentences, sum up what you've done for the week

**Comment by MiguelGibo** [comment]
Wrote 2 new issues (#40 & #41) and contributed to discussions on both GitHub and in team meetings.

**Comment by AlejandroClaudio** [comment]
I didn't get to work much on the project, other than participating in polls and a group meeting about features and them features being divided between team members.

**Comment by estefania-2817** [comment]
I did not contribute to the project this week. I did help with the decision making answering through polls. I attended the group meeting and selected the feature I want to work on.

**Comment by GabyMarr** [comment]
I did not contribute to the project this week despite attending meeting and answering polls on features/Tech Stack.

**Comment by AlexandraFuentes** [comment]
I did contribute to the project this week; I made the figma design, created logo and name for project, created discussions/issues and closed issues, did leardership/managerial tasks, and participated in the meeting.

**Comment by KennethSepu1** [comment]
I did not contribute to the project this week but I did attend the meetings and answered different polls and discussions.

**Comment by Ingrid1089** [comment]
I did not work on the project this week. But, I participated in polls and attended the group meetings.

**Comment by WilsonMorales8** [comment]
I did not directly work on the project. I helped in decision making through the means of polls and group meetings.

**Comment by marymarmaldonado** [comment]
Started the Svelte tutorial, attended the Thursday meeting, and voted on polls.

**Comment by LuisJCruz** [comment]
I did not work directly on the project this week since we have been getting prepared and dividing the teams. Currently learning Sveltle to start work on the homepage next week.

**Comment by Ojani** [comment]
I closed issues #52, #45, and #47. All of these are basically project setup so that the others can soon get started working on their features and not worry about setup. In summary, I made the [documentation guide](https://sistemaupr-my.sharepoint.com/:f:/g/personal/ojani_figueroa_upr_edu/Eqlo9P4D4UZHg4bHjCxJs90BfMWxwdf-nHmwlT38ll9_Vw?e=xrcjdD), Made the project's skeleton, and added the instructions on the README to get started with development.

**Comment by Adrianortiz-PR** [comment]
Closed discussion post on the technology stack, closed issues #54. Opened issue #53 for login team. Lead team meetings and was assigned Team leader

**Comment by EdzelOrtiz** [comment]
For this week, I was working on thinking of how to manage the many tags that would exist. I brought the idea to see if we created different windows in which the different tags will appear depending the Faculty chosen by the user, so that it does not have tags that the user is not interested in.

**Comment by cseba11** [comment]
I began working through the Svelte tutorial to get familiar with the framework, participated in the team’s Thursday meeting where we discussed project updates and upcoming tasks, and also cast my votes in the polls for team decisions.

**Comment by AlexisSantiago21** [comment]
I completed issue #55 and participated in the weekly meeting.

**Comment by kevin-d-lopez** [comment]
I started to learn svelte and added to the tags doc for task #57.

**Comment by Kiarancole** [comment]
I started to learn Svelte through it's tutorial and contributed to the Profile Event Turner tag documentation in issue #57, as well as attending the weekly meetings and participating in discussions/polls.
